src/components/Week/Week.js

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import styled from 'styled-components';
 
 import activityPropTypeShape from '../../prop-types/activity';
 import Day from '../Day/Day';
 import { daysOfTheWeek } from '../../constants/dates';
 import { getActivityItems, getActivitiesForDay } from '../../store/activities/selectors';
+import { spacing } from '../../styles/base/base';
+
+const StyledWeekWrapper = styled.div`
+  @media (min-width: ${spacing.desktopBreakpoint}) {
+    align-items: stretch;
+    display: flex;
+    height: 80vh;
```

Comment:
this height seems to be more legit, but still, I would try to avoid that also in this place.